pkg/lib/machine-auth-failed.html

```diff
@@ -3,7 +3,7 @@ <h4 class="modal-title" translate="yes">Authentication Failed</h4>
 </div>
 <div class="modal-body">
     {{#supported}}
-    <p translate="yes">Cockpit was unable to log into {{#strong}}{{host}}{{/strong}}. To use this machine with cockpit you will need to enable one of the following authentication methods in the sshd config on {{#strong}}{{host}}{{/strong}}:</p>
+    <p translate="yes">Unable to log in to {{#strong}}{{host}}{{/strong}}. To use this machine with cockpit you will need to enable one of the following authentication methods in the sshd config on {{#strong}}{{host}}{{/strong}}:</p>
```

Comment:
While we are changing the string, the second "with cockpit" should IMHO be dropped as well. Also, you renamed machine → host in other places (and I agree). How about "To connect to this host you will need ..."

Reply:
Yes, done.